plugins/module_utils/network/sonic/facts/facts.py

```diff
@@ -98,7 +99,8 @@
     bfd=BfdFacts,
     copp=CoppFacts,
     route_maps=Route_mapsFacts,
-    stp=StpFacts
+    stp=StpFacts,
+    sFlow=SflowFacts
```

Comment:
you might have to update `sFlow` to `sflow` here if you address my first comment from argspec/facts/facts.py